Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/Scanner/ScanningViewController.swift

```diff
@@ -71,6 +71,7 @@ public final class ScanningViewController<CaptureSession: GDSCommon.CaptureSessi
     
     public override func viewDidLoad() {
         super.viewDidLoad()
+        setBackButtonTitle()
```

Comment:
This should be handled by `BaseViewController` so shouldn't need to be set separately?

Reply:
Hmm, currently it does not pick up the welsh translation

Reply:
Okay I think I know the issue, I believe it is looking for the strings in main, but there is no option to pass in that we need to look in module for Wallet. Would need to be able to pass the string through to that setBackButton function, which would mean adding a backButtonTitle to BaseViewController, which is a breaking change.

Reply:
I think something like this in the `setBackButton` method should sort that out:

```
let bundle: Bundle
        if Bundle.main != nil {
            bundle = .main
        } else {
            bundle = .module
        }
```

Reply:
@amopatel1234 @robinphillips updated now!

Reply:
Don't you need to add the `localizable.strings` file into the `GDSCommon` module too for this to work?